Neos.Utility.ObjectHandling/Classes/TYPO3/Flow/Reflection/ObjectAccess.php

```diff
@@ -97,13 +97,10 @@ protected static function getPropertyInternal($subject, $propertyName, $forceDir
     {
         if ($subject === null) {
             return null;
-        }
-        if (is_array($subject)) {
-            if (array_key_exists($propertyName, $subject)) {
-                $propertyExists = true;
-                return $subject[$propertyName];
-            }
-            return null;
+        } elseif (is_array($subject)) {
+            $propertySet = isset($subject[$propertyName]);
+            $propertyExists = $propertySet || array_key_exists($propertyName, $subject);
```

Comment:
It would suffice, but `isset()` is much faster than `array_key_exists()`.

`$propertyExists` is passed by reference to the method ;)